src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/health/HealthInfo.kt

```diff
@@ -14,3 +17,11 @@ class HealthInfo(buildProperties: BuildProperties) : HealthIndicator {
 
   override fun health(): Health = Health.up().withDetail("version", version).build()
 }
+
+@Component
+class MyInfoContributor(@Value("\${application.product-id}") private val productId: String? = null) : InfoContributor {
```

Comment:
would have expected this to be `productId`?  Also shouldn't it be not nullable if you want people to enter it?